web/controllers/organization_membership_controller.ex

```diff
@@ -19,19 +21,12 @@ defmodule CodeCorps.OrganizationMembershipController do
     render(conn, "index.json-api", data: memberships)
   end
 
-  def show(conn, %{"id" => id}) do
-    membership =
-      OrganizationMembership
-      |> preload([:organization, :member])
-      |> Repo.get!(id)
-
-    render(conn, "show.json-api", data: membership)
+  def show(conn, %{"id" => _id}) do
+    render(conn, "show.json-api", data: conn.assigns.organization_membership)
```

Comment:
This `conn.assigns.organization_membership` reads odd to me. Can you explain that?